pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -19,6 +19,7 @@
 from pandas.compat.numpy import np_datetime64_compat
 from pandas.errors import PerformanceWarning
 
+import pandas as pd
```

Comment:
import are all crazy here, instead of adding can you just  do
```
from pandas import DatetimeIndex, date_range, Series, read_pickle
```

and remove the other references

Reply:
Done